config.yaml

```diff
@@ -0,0 +1,2 @@
+# include version specification in configuration files to help with parsing and schema evolution.
+version: 0.0.1
```

Comment:
What would be the use case to bump the "patch number"?

I am used to have "patch" when versioning the config files. Usually it is only "major" or "major.minor" depending on how strict is the parser.

Reply:
Also I thinj it should be named `scheme-version`. `version` suggests that this is the version of the config.